ReadMe.md

```diff
@@ -23,10 +29,6 @@ All documentation for the toolkit is hosted on [Microsoft Docs](https://docs.mic
 
 Want to see the toolkit in action before jumping into the code? Download and play with the [Windows Community Toolkit Sample App](https://www.microsoft.com/store/apps/9nblggh4tlcq) from the Store.
 
-## 🧪 Windows Community Toolkit Labs 💻
```

Comment:
We should probably still point to labs (though move it up) as new features are being developed there as well still.